content/secrets/04-walkthrough-step-01/copilot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+In this section, it is important to match the names as described in order for the tutorial to work (so that it matches the format in the repository)
+
+Application Name: `ecsworkshop`
+
+Workload Type: `Load Balanced Web Service`
+
+Service Name: `todo-app`
+
+Deploy local test environment: `yes`
+
+
+```bash
+copilot init
```

Comment:
In the codebase github.com/mptaws/secretecs, can you replace the usage of "Master" with "Main". I caught a few references in the addons db.yml